src/appComponents/sponsors/SponsorsController.js

```diff
@@ -26,13 +28,15 @@ sponsorsModule.controller('SponsorsController',
         sc.Sponsors = $sessionStorage.sponsors;
 
         if (sc.Sponsors.length === 0) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 8.](https://www.codacy.com/app/triplez/open-event-webapp/file/2432951250/issues/source?bid=3233857&fileBranchId=3238420#l30)